bm2/structs.h

```diff
@@ -0,0 +1,19 @@
+//structs of bm2
```

Comment:
In my repository this file is called "strucs.h", so this will create a duplicate. But I'll fix that after merge.